releasenotes/notes/initial_state_draw_parameter-f360ac4e998ee944.yaml

```
@@ -0,0 +1,32 @@
+---
+other:
```

I would probably do this as an upgrade note not other. Since this is a change in behavior for the `draw()` methods and if users want to retain the behavior of having an initial state in their circuit visualizations they need to set a kwarg now. We probably should also split the new flag as a separate feature note.

```suggestion
upgrade:
```